salt/_modules/caasp_etcd.py

```diff
@@ -172,10 +170,12 @@ def get_endpoints(with_id=False, skip_this=False, skip_removed=False, port=ETCD_
         member_endpoint = 'https://{}:{}'.format(name, port)
         if with_id:
             member_endpoint = "{}={}".format(node_id, member_endpoint)
-        etcd_members_lst.append(member_endpoint)
+            etcd_members_lst.append(member_endpoint)
+        else:
+            etcd_members_lst.append(member_endpoint)
```

Comment:
I think this should be left as it was, as `etcd_members_lst.append(member_endpoint)` will be done anyway.

Reply:
I just realized that it was already `dedented` one level - I must have run into some spurious error, because you are obviously right ...